linode/manager
# fix: [M3-7038] - Add Firewall and Linode Configuration success toast notifications

## Description 📝
The Firewall and Linode Configuration toast notifications did not have a success message, causing no toast notifications when an action was successful.

## Major Changes 🔄
- Added success notifications for
   - Firewall notifications (enable, disable, create, delete, device add, device remove)
   - Linode configurations (create, delete, update)
- Updated toast notification messages for consistency

## Preview 📷

<details>
<summary>Before</summary>
<video src="https://github.com/linode/manager/assets/139489745/3a840a5a-5444-47e0-8ea9-5411e4aa58c0" />
</details>

<details>
<summary>After</summary>

![Screenshot 2023-09-27 at 10 30 38 AM](https://github.com/linode/manager/assets/139489745/7f276db5-c618-4429-9151-42c280fea385)

![Screenshot 2023-09-27 at 10 29 51 AM](https://github.com/linode/manager/assets/139489745/7f6fa81e-435c-4965-8c0b-b7cc81ab0249)

![Screenshot 2023-09-27 at 10 30 05 AM](https://github.com/linode/manager/assets/139489745/ec6d674b-9988-45e8-abaa-77ed7467ccde)

![Screenshot 2023-09-27 at 10 31 52 AM](https://github.com/linode/manager/assets/139489745/3c49cc16-d110-4d20-a951-f9c71a40ffd3)

![Screenshot 2023-09-27 at 10 32 09 AM](https://github.com/linode/manager/assets/139489745/68ef01d5-5d15-4c48-9a46-531e23ff2471)

![Screenshot 2023-09-27 at 10 32 41 AM](https://github.com/linode/manager/assets/139489745/b0089123-3f82-49ca-9b3d-6ce014f58bff)

![Screenshot 2023-09-27 at 10 30 22 AM](https://github.com/linode/manager/assets/139489745/c8ae2b98-d968-42d1-a120-4eb40897b7f0)

![Screenshot 2023-09-27 at 10 36 05 AM](https://github.com/linode/manager/assets/139489745/a04fb822-e3f8-48b4-9817-dbc5c7750502)

</details>

## How to test 🧪

*** Note: The toast notifications take a couple seconds to pop up.

Navigate to the firewall landing page (/firewalls) and ensure:
- [ ] Creating a Firewall triggers a successful toast notification
- [ ] Disabling a Firewall triggers a successful toast notification
- [ ] Enabling a Firewall triggers a successful toast notification
- [ ] Deleting a Firewall triggers a successful toast notification

Navigate to the linodes tab within any firewall details page (/firewalls/[firewall_ID]/linodes) and ensure:
- [ ] Adding a Linode triggers a successful toast notification
- [ ] Deleting a Linode triggers a successful toast notification

Navigate to the config tab within any linode details page (/linodes/[linode_id]/configurations) and ensure:
- [ ] Adding a Linode configuration triggers a successful toast notification
- [ ] Deleting a Linode configuration triggers a successful toast notification
- [ ] Editing a Linode configuration triggers a successful toast notification
